components/Idea/Idea.js

```diff
@@ -1,16 +1,44 @@
-export default function Idea({ id, date, text, handleDeleteIdea }) {
+import IdeasMoveArea from "./../IdeasMoveArea";
+
+export default function Idea({
+  id,
+  date,
+  text,
+  projects,
+  isMovingIdea,
+  handleDeleteIdea,
+  handleMoveIdeaToggle,
+  selectedProjectForIdea,
+  handleSelectedProjectForIdea,
+  handleAddIdeaToProject,
+}) {
   return (
     <li className="ideasList__newIdea">
       <p>{text}</p>
       <article className="ideasList__newIdea--bottom">
         <small>{date}</small>
+
         <button
-          className="ideasList__newIdea--deleteButton"
+          className="primaryButton__delete"
           onClick={() => handleDeleteIdea(id)}
         >
           Löschen
         </button>
-      </article>
+        <button
+          className="primaryButton"
+          onClick={() => handleMoveIdeaToggle(id)}
+        >
+          Verschieben
+        </button>
+      </article>{" "}
+      {isMovingIdea[id] ? (
+        <IdeasMoveArea
+          projects={projects}
+          selectedProjectForIdea={selectedProjectForIdea}
+          handleSelectedProjectForIdea={handleSelectedProjectForIdea}
+          handleAddIdeaToProject={handleAddIdeaToProject}
+        />
+      ) : null}
```

Comment:
You don't need the ternary operator if you just want to render nothing if the condition is not met:
```suggestion
      {isMovingIdea[id] && (
        <IdeasMoveArea
          projects={projects}
          selectedProjectForIdea={selectedProjectForIdea}
          handleSelectedProjectForIdea={handleSelectedProjectForIdea}
          handleAddIdeaToProject={handleAddIdeaToProject}
        />
      )}
```